src/Mockingbird/Utils/Default.swift

```diff
@@ -24,10 +25,51 @@ final class Default {
     public enum Folder {
 
         static var main = "/Users/" + NSUserName() + "/.mockingbird"
-        static var capture: String { main + "/capture" }
-        static var data: String { main + "/data" }
         static var mitm: String { main + "/mitmproxy" }
-        static var record: String { main + "/record" }
-        static var test: String { main + "/test" }
+        static var capture: String { workingDirectory + "/capture" }
+        static var record: String { workingDirectory + "/record" }
+        static var data: String { workingDirectory + "/data" }
+        static var test: String { workingDirectory + "/test" }
+
+        static var workingDirectory = ""
+
+        static var savedWorkingDirectory: String {
+
+            get {
+
+                guard let file = try? File(path: "\(Default.Folder.main)/configurations.json"),
+                      let fileData = try? file.readAsString(),
+                      let startRange = fileData.range(of: "=")?.upperBound else {
+
+                    return "/Users/" + NSUserName() + "/.mockingbird"
+                }
+
+                return String(fileData.suffix(from: startRange))
+            }
+
+            set {
+
+                if let configurationsFile = try? File(path: "\(Default.Folder.main)/configurations.json") {
+
+                    let configurations = Configuration(dataFolder: newValue)
```

Comment:
can we move those common codes inside if/else to a common function?

Reply:
Moved to a new static function!